release-workflow.md

```diff
@@ -0,0 +1,41 @@
+# Release Workflow
+
+## Trigger
+
+Schedule (every 5 minutes) or push to `master` branch.
+
+## Development
+
+With `PRODUCTION=0` in environment, the script doesn't publish but outputs the commands that would be run. A publish is only triggered when `PRODUCTION=1` is set.
+
+Entry point to test development is `yarn vsce:check <channel>`, where `channel=dev|latest`.
+
+## Steps
+
+1. Defined in the file `.github/workflows/publish.yml`
+1. `yarn vsce:check <channel>` is run twice with "dev" and "latest" as channels respectively. Internally, it calls `check-update.sh <channel>`
+
+Note the both `dev` workflow and `latest` workflow call the same scripts, the channel variable acts as a workflow discriminator as both workflows have slight differences, now we list the workflow steps in details:
+
+### Dev workflow
+
+1. `yarn vsce:check dev` calls `check-update.sh dev`
+1. `check-update.sh` sets up the repo with Prismo bot as the user, all commits in the remainder of this workflow are attributed to Prismo.
```

Comment:
just a little typo
```suggestion
1. `check-update.sh` sets up the repo with Prismo bot as the user, all commits in the remainder of this workflow are attributed to Prisma.
```

Reply:
Oh noes, the bot name is Prismo indeed - https://github.com/prisma-bot